### DISCUSSION TITLE
Skip empty folders in "Browse code" page and paths

### DISCUSSION THREAD

**Discussion by AyakaAgo** [community/community#59361] [Mobile] [answered]
### Select Topic Area

Product Feedback

### Body

Hi there! 👋

In web we can skip empty folders and navigate up to any parent folder by select the path
![Screenshot_2023-06-27-01-01-08-382_mark via](https://github.com/community/community/assets/20831549/ae9383bf-3b33-4b43-bb1c-6423b199fe7d)

but in app we can only navigate one by one
![Screenshot_2023-06-27-01-01-00-525_com github android](https://github.com/community/community/assets/20831549/afbdbf3d-d53d-4f56-babb-b829929bd662)

1. any option to skip empty folders
2. add file path bar in "Browse code" page
![IMG_20230627_011451](https://github.com/community/community/assets/20831549/54207a9d-3a60-4256-bc94-3807bbe0a42c)

Github Mobile v1.119.0-beta (10135)

**Comment by stevepopovich** [comment, marked as answer]
Hi @AyakaAgo thanks for the feedback. I passed this onto the team for prioritization. Thanks!
